docs/src/pages/index.tsx

```diff
@@ -6,29 +6,31 @@ function HomepageHeader() {
   return (
     <header>
       <section className="text-center m-6 p-12 border border-red-400 rounded-[50px] bg-gray-100 dark:bg-immich-dark-gray">
-        <img src="img/immich-logo.svg" className="md:h-24 h-12 mb-2" />
-        <h1 className="md:text-6xl font-immich-title mb-10 text-immich-primary dark:text-immich-dark-primary">
-          IMMICH
+        <img src="img/immich-logo.svg" className="md:h-24 h-12 mb-2" alt="Immich logo" />
+        <h1 className="md:text-6xl font-immich-title mb-10 text-immich-primary dark:text-immich-dark-primary uppercase">
+          Immich
         </h1>
         <div className="font-thin sm:text-base md:text-2xl my-12 sm:leading-tight">
-          <p className="mb-1">SELF-HOSTED BACKUP SOLUTION </p>
-          <p className="mb-1">FOR PHOTOS AND VIDEOS</p>
-          <p className="mb-1">ON MOBILE DEVICE</p>
+          <p className="mb-1 uppercase">
+            Self-hosted backup solution <span class="block"></span>
+            for photos and videos <span class="block"></span>
```

Comment:
This is a react component so I think you need to use className. Try to run the npm checks before pushing to make sure it compiles and passes linting.